packages/flutter_tools/lib/src/base/utils.dart

```diff
@@ -43,7 +55,7 @@ class BotDetector {
 
         // Properties on Flutter's Chrome Infra bots.
         || platform.environment['CHROME_HEADLESS'] == '1'
-        || platform.environment.containsKey('BUILDBOT_BUILDERNAME'));
+        || platform.environment.containsKey('BUILDBOT_BUILDERNAME');
```

Comment:
While we're here, can we add `|| platform.environment.containsKey('SWARMING_TASK_ID')`?  

I strongly suspect `BUILDBOT_BUILDERNAME` was used previously on Chrome infra, but it's definitely not getting set on LUCI today.  AFAICT we're not excluding LUCI runs from metrics reports.

Reply:
(It should be safe to leave the current check in place though)

Reply:
Done.